core/util.go

```diff
@@ -316,6 +317,26 @@ func UniqueLowerNames(names []string) (unique []string) {
 	return
 }
 
+// UniqueLowerACMEIdentifiers returns the set of all unique ACME identifiers in
+// the input after all of them are lowercased. The returned identifier values
+// will be in their lowercased form.
+func UniqueLowerACMEIdentifiers(identifiers []identifier.ACMEIdentifier) (unique []identifier.ACMEIdentifier) {
```

Comment:
This is painting the bikeshed bright fuchsia but... maybe `core.NormalizeIdentifiers`? "UniqueLowerACMEIdentifers" is quite long, and "lower" doesn't apply to IPAddrs...